src/blocks/recommendations/components/parsely-recommendations-list-item.tsx

```diff
@@ -11,15 +11,19 @@ import { Recommendation } from '../models/Recommendation';
 interface ParselyRecommendationsListItemProps {
 	imageAlt: string;
 	imagestyle: string;
+	openlinksinnewtab: boolean;
 	recommendation: Recommendation;
 	showimages: boolean;
 }
 
 const getImageForLink = ( imagestyle: string, imageUrl: string, thumbUrlMedium: string ) => imagestyle === 'original' ? imageUrl : thumbUrlMedium;
+const getLinkTarget = ( openlinksinnewtab: boolean ) => Boolean( openlinksinnewtab ) === true ? '_blank' : '_self';
```

Comment:
What do you think to havesomething like:
```
const getLinkObject = ( openlinksinnewtab: boolean ) => Boolean( openlinksinnewtab ) === true ? { target: '_blank' , rel: 'noopener' } : { target: '_self', rel: '' };
```

Reply:
Thank you, this led me towards the right way to do this. In my previous attempt I was returning strings and this wasn't working for me.